app/services/integrations/destroy_service.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Integrations
+  class DestroyService < BaseService
+    def destroy(id:)
```

Comment:
What about using the `call` approach, and passing the arguments in the initializer?